subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -119,10 +119,11 @@ menu "Advanced features"
 	visible if BT_CTLR_ADVANCED_FEATURES
 
 config BT_LL_SW_SPLIT_LLCP_LEGACY
-	bool "Legacy software-based BLE Link Layer"
+	bool "Legacy software-based BLE Link Layer [EXPERIMENTAL]"
 	default y
 	help
 	  Use Zephyr legacy software BLE Link Layer ULL LLL split implementation.
+	  When not selected the experimental refactored LLCP implementation is used.
```

Comment:
This is LLCP related, the title and help text are incorrect.

The new refactor LLCP is to be experimental.

Reply:
This is DNM and also superseded by https://github.com/zephyrproject-rtos/zephyr/pull/39736